src/fetchData.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable no-redeclare */
+/* global, fetch document */
+
+import loadDOM from "./loadDOM";
+
+const fetchData = () => {
+  loadDOM();
+  let location = document.getElementById("city");
+  let city = location.value;
+  let fetchUrl = `https://api.openweathermap.org/data/2.5/weather?q=${city}&appid=fa1c7f10e6e75b7dc652e684202b5bd1`;
+  let displayLocation = document.querySelector(".location");
+
+  fetch(fetchUrl, { mode: "cors" })
+    .then(response => {
+      return response.json();
+    })
+    .then(response => {
+      const cityName = response.name;
+      const { temp, humidity, pressure } = response.main;
+      const { deg } = response.wind;
+      const { country } = response.sys;
+
+      displayLocation.textContent = `${cityName}, ${country}`;
```

Comment:
This DOM related logic should be in a separate DOM module. This module should only focus on fetching the data.